public/docs/ts/latest/guide/architecture.jade

```diff
@@ -1,190 +1,218 @@
   1. [Metadata](#metadata)
-  1. [Data Binding](#data-binding)
-  1. [Directive](#directive)  
-  1. [Service](#service)
-  1. [Dependency Injection](#dependency-injection)
+  1. [Data binding](#data-binding)
+  1. [Directives](#directives)
+  1. [Services](#services)
+  1. [Dependency injection](#dependency-injection)
 
-  Learn these eight and we're on our way.
+  Learn these, and we're on our way.
 
 .l-sub-section
-  :marked
-      The code referenced in this chapter is available as a [live example](/resources/live-examples/architecture/ts/plnkr.html).
-<a id="module"></a>
+  p The code referenced in this chapter is available as a #[+liveExampleLink2()].
+
 .l-main-section
 :marked
-  ## The Module
+  ## Modules
 figure
   img(src="/resources/images/devguide/architecture/module.png" alt="Component" align="left" style="width:240px; margin-left:-40px;margin-right:10px" )
 :marked
-  Angular apps are modular. 
-  
+  Angular apps are modular.
+
   In general we assemble our application from many **modules**.
-  
+
   A typical module is a cohesive block of code dedicated to a single purpose.
   A module **exports** something of value in that code, typically one thing such as a class.
   <br clear="all"><br>
-.l-sub-section
-  :marked
-     ### Modules are optional
-     We highly recommend modular design. TypeScript has great support for ES2015 module syntax and our chapters assume we're taking a modular
-     approach using that syntax. That's why we list *Module* among the basic building blocks.
-     
-     Angular itself doesn't require a modular approach nor this particular syntax. Don't use it if you don't want it.
-     Each chapter has plenty to offer after you steer clear of the `import` and `export` statements.
-     
-     Find setup and organization clues in the JavaScript track (select it from the combo-box at the top of this page) 
-     which demonstrates Angular 2 development with plain old JavaScript and no module system. 
-:marked
-  Perhaps the first module we meet is a module that exports a *component* class. 
+
+block dart-terminology
+  //- N/A
+
+block modules-are-optional
+  .l-sub-section
+    :marked
+       ### Modules are optional
+       We highly recommend modular design. TypeScript has great support for ES2015 module syntax and our chapters assume we're taking a modular
+       approach using that syntax. That's why we list *Module* among the basic building blocks.
+
+       Angular itself doesn't require a modular approach nor this particular syntax. Don't use it if you don't want it.
+       Each chapter has plenty to offer after you steer clear of the `import` and `export` statements.
+
+       Find setup and organization clues in the JavaScript track (select it from the combo-box at the top of this page)
+       which demonstrates Angular 2 development with plain old JavaScript and no module system.
+
+:marked
+  Perhaps the first module we meet is a module that exports a *component* class.
```

Comment:
again with the "export"

In Dart (if we need to keep this section), I'd use something like "defines" instead.